vnext/Microsoft.ReactNative.Cxx/ReactNotificationService.h

```diff
@@ -0,0 +1,298 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#ifndef MICROSOFT_REACTNATIVE_REACTNOTIFICATIONSERVICE
+#define MICROSOFT_REACTNATIVE_REACTNOTIFICATIONSERVICE
+
+#include "ReactDispatcher.h"
+#include "ReactPropertyBag.h"
+
+namespace winrt::Microsoft::ReactNative {
+
+// Encapsulates the IReactPropertyName and the notification data type
+template <class T>
```

Comment:
consider simplifying this by just making this an alias instead of inheriting from it

Reply:
I do not see how it can be done. The alias will not allow me to access the template type. Inheritance here is cheap: we inherit from a non-template class.

---
In reply to: [427649718](https://github.com/microsoft/react-native-windows/pull/4953#discussion_r427649718) [](ancestors = 427649718)